test/AutoRest.TestServer.Tests/body-dictionary.cs

```diff
@@ -498,11 +448,10 @@ public class BodyDictionary : TestServerTestBase
         });
 
         [Test]
-        [Ignore("Null responses: https://github.com/Azure/autorest.csharp/issues/289")]
-        public Task GetDictionaryNull() => Test(async (host, pipeline) =>
+        public Task GetDictionaryNull() => Test((host, pipeline) =>
         {
-            var result = await new DictionaryClient(ClientDiagnostics, pipeline, host).GetNullAsync();
-            Assert.IsNull(result.Value);
+            // Empty response body
+            Assert.ThrowsAsync(Is.InstanceOf<JsonException>(), async () => await new DictionaryClient(ClientDiagnostics, pipeline, host).GetNullAsync());
```

Comment:
Is null returned here or an empty response?

Reply:
Empty response. No test server tests return `null`. All the test server test names that either say `NotProvided` or `Null` return an empty body. They always look like this:
```js
res.status(200).end();
```